data/update/postgres/dbupdate-7.4.0-to-7.5.0.sql

```diff
@@ -24,3 +24,9 @@ ADD CONSTRAINT ezcontentclass_attribute_ml_lang_fk
   REFERENCES ezcontent_language (id)
   ON DELETE CASCADE
   ON UPDATE CASCADE;
+
+--
+-- EZP-30139: As an editor I want to hide and reveal a content item
+--
+
+ALTER TABLE ezcontentobject ADD is_hidden boolean DEFAULT 0 NOT NULL;
```

Comment:
This is not going to work.

```suggestion
ALTER TABLE ezcontentobject ADD is_hidden boolean DEFAULT false NOT NULL;
```

Please actually test all your upgrade scripts :trollface: 

But of course a little bit my fault also, because when requesting change here, I forgot to mention that default value needs alignment ;)